lib/v2/cntv/column.js

```diff
@@ -0,0 +1,29 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.column;
+    const limit = ctx.params.limit ?? 25;
```

Comment:
`ctx.query.limit`. 
`ctx.params.limit` has no impact on the items count as it will be always 25 as `ctx.params.limit` is always undefined.

Reply:
Thx, it's so helpful.